interfaces/snapshot_downloader/external_downloader.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
```

Comment:
This file should not be put into this directory directly. It needs to first be submitted into `ledgerwatch/interfaces` repo, and then (follow instructions in the `README.md` there), a new version of interfaces needs to be pulled into TG